README.md

```diff
@@ -45,20 +45,18 @@ Import `@runno/runtime` in whatever place you'll be using the runno elements.
 The simplest is in your entrypoint file (e.g. `main.ts` or `index.ts`).
 
 ```js
-import '@runno/runtime';
+import "@runno/runtime";
 ```
 
 Once you've imported them you can use runno elements on the page.
 
 ```html
-<runno-run runtime="python" editor controls>
-print('Hello, World!')
-</runno-run>
+<runno-run runtime="python" editor controls> print('Hello, World!') </runno-run>
 ```
 
 For the code to run though, you'll need to set some HTTP headers:
 
-```HTTP
+```
```

Comment:
This did [previously provide some syntax highlighting](https://github.com/taybenlor/runno/blob/baf230b378aee44735f3557e93d3ca506030e57c/packages/runtime/README.md#quickstart). I didn't even know you could syntax highlight HTTP headers until I saw this